tests/test_api_requestor.py

```diff
@@ -324,9 +332,9 @@ async def test_empty_methods_streaming_response_async(
             http_client_mock_streaming_async.assert_requested(
                 meth, post_data=post_data
             )
-            assert isinstance(resp, StripeStreamResponse)
+            assert isinstance(resp, StripeStreamResponseAsync)
 
-            assert resp.io.getvalue() == b"thisisdata"
+            assert b"".join([x async for x in resp.stream()]) == b"thisisdata"
```

Comment:
Do you think it would be helpful to add a `read()` method that does this for the user?

Reply:
Hmm, I'll consider in a follow-up.